src/styles.scss

```diff
@@ -32,10 +64,20 @@ markdown h2 {
   border-bottom: 1px solid white;
 }
 
-ul {
-  padding-left: 20px;
+mat-card {
+  background-color: rgb(29, 48, 95) !important;
 }
 
-li {
-  margin-bottom: 1rem;
+mat-dialog-container {
+  background-color: rgb(29, 48, 95) !important;
 }
+
+mat-dialog-content {
+  ul {
+    padding-left: 20px;
+  }
+  
+  li {
+    margin-bottom: 1rem;
+  }
+}
```

Comment:
This can be moved to the dialog component's SCSS file.

Reply:
I tried putting them in dialog.component.scss, but it didn't work for some reason.

Reply:
And without the mat-dialog-content wrapper, same thing.

Reply:
Can use `::ng-deep` to make the rule propagate to the content within `innerHTML`. ex. `::ng-deep ul { ... }`

Reply:
Oh, there we go.